client/widgets/MiscWidgets.h

```diff
@@ -123,7 +123,7 @@ class LRClickableAreaWTextComp: public LRClickableAreaWText
 {
 public:
 	int baseType;
-	int bonusValue, type;
+	int bonusValue;
```

Comment:
And you of course can't just rename it just here without changing name in 14 (more probably?) places in the code. Anyway I don't see the point.